src/Fable.Cli/Util.fs

```diff
@@ -93,7 +93,18 @@ type GlobalParams private (verbosity, forcePkgs, fableLibraryPath, workingDir) =
 
     member __.Verbosity: Fable.Verbosity = _verbosity
     member __.ForcePkgs: bool = _forcePkgs
-    member __.FableLibraryPath: string = _fableLibraryPath
+    member __.FableLibraryPath (isTypeScript : bool) : string =
```

Comment:
There is also still some strange implementation/hacks that are here just for the prototype. We should try to clean this before releasing a "test" version advertising beta support for TypeScript.